rules/ai-pair-programming/rule.md

```diff
@@ -0,0 +1,30 @@
+---
+type: rule
+title: Do you use AI pair programming?
+uri: ai-pair-programming
+authors:
+  - title: Piers Sinclair
+    url: https://www.ssw.com.au/people/piers-sinclair
+created: 2022-09-20T05:29:57.151Z
+guid: 2cca5396-bb31-4ff6-b7b8-a613c3362f72
+---
+Too often, developers are writing a line of code, and they forget that little bit of syntax they need to do what they want. In that case, they usually end up googling it to find the documentation and then copy and paste it into their code. That process is a pain because it wastes valuable dev time. 
+
+Not to worry, AI pair programming is here to save the day! 
+            
+<!--endintro-->
+
+New tools like [GitHub Copilot](https://github.com/features/copilot) provide devs with fast access to the syntax they need in moments. It might sound like it's too good to be true, but in reality you can do so much with these tools.
```

Comment:
```suggestion
New tools like [GitHub Copilot](https://github.com/features/copilot) provide devs with fast access to potentially fully worked solutions they need in moments. It might sound like it's too good to be true, but in reality you can do so much with these tools.
```